packages/babel-helper-create-class-features-plugin/src/fields.ts

```diff
@@ -669,6 +669,16 @@ const thisContextVisitor = traverse.visitors.merge([
   environmentVisitor,
 ]);
 
+const innerReferencesVisitor = {
+  ReferencedIdentifier(path, state) {
+    const { name } = path.node;
+    if (path.scope.bindingIdentifierEquals(name, state.innerBinding)) {
+      state.needsClassRef = true;
+      path.replaceWith(t.cloneNode(state.classRef));
```

Comment:
Since we only changing name, may be we can just modify `path.node.name` instead of requeueing the AST node?

Reply:
Aha. I didn't know that. Thank you for your advice.